src/pages/outwards-return/index.js

```diff
@@ -0,0 +1,162 @@
+import { useContext } from 'react'
+import Table from 'src/components/Shared/Table'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useWindow } from 'src/windows'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useError } from 'src/error'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { getStorageData } from 'src/storage/storage'
+import { RemittanceOutwardsRepository } from 'src/repositories/RemittanceOutwardsRepository'
+import OutwardsReturnForm from './Forms/OutwardsReturnForm'
+import GridToolbar from 'src/components/Shared/GridToolbar'
+import { SystemFunction } from 'src/resources/SystemFunction'
+import { useDocumentTypeProxy } from 'src/hooks/documentReferenceBehaviors'
+
+const OutwardsReturn = () => {
+  const { getRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack: stackError } = useError()
+  const { stack } = useWindow()
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50, params } = options
+
+    const response = await getRequest({
+      extension: RemittanceOutwardsRepository.OutwardsReturn.qry,
+      parameters: `_startAt=${_startAt}&_pageSize=${_pageSize}&_params=${params || ''}&filter=`
```

Comment:
qryOW doesn't request any of these `_startAt=${_startAt}&_pageSize=${_pageSize}&_params=${params || ''}`